h5py/_hl/files.py

```diff
@@ -123,6 +124,7 @@ def make_fapl(driver, libver, rdcc_nslots, rdcc_nbytes, rdcc_w0, locking,
         # we default to earliest
         low, high = h5f.LIBVER_EARLIEST, h5f.LIBVER_LATEST
     plist.set_libver_bounds(low, high)
+    plist.set_alignment(alignment_threshold, alignment_interval)
```

Comment:
However, my thoughts were the same as yours, that the likeliness of their override was minor.

Reply:
Yup, I'm happy enough with this. I just figured that I'd leave a note here in case someone in 10 years has to try to work out what happened. :laughing: